pkg/app/metadata.json

```diff
@@ -0,0 +1,963 @@
+{
+  "acl": {
+    "create": {
+      "examples": [{
+        "cmd": "fastly acl create --name robots --version latest",
+        "title": "Create a new ACL attached to the specified service version"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/acls/acl/#create-acl"]
+    },
+    "delete": {
+      "examples": [{
+        "cmd": "fastly acl delete --name robots --version latest",
+        "title": "Delete an ACL from the specified service version"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/acls/acl/#delete-acl"]
+    },
+    "describe": {
+      "examples": [{
+        "cmd": "fastly acl describe --name robots --version latest",
+        "title": "Retrieve a single ACL by name for the version and service"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/acls/acl/#get-acl"]
+    },
+    "list": {
+      "examples": [{
+        "cmd": "fastly acl list --version latest",
+        "title": "List ACLs for the version and service"
```

Comment:
"version and service" is very awkward language and "service version" should be favoured across the board (I know "version and service" is a phrase that exists in the API docs, we should get around to fixing that too).